internal/gcsx/bucket_manager_test.go

```diff
@@ -91,7 +91,32 @@ func (t *BucketManagerTest) TestSetUpBucketMethod() {
 	bm.config = bucketConfig
 	bm.gcCtx = ctx
 
-	bucket, err := bm.SetUpBucket(context.Background(), TestBucketName)
+	bucket, err := bm.SetUpBucket(context.Background(), TestBucketName, false)
+
+	ExpectNe(nil, bucket.Syncer)
+	ExpectEq(nil, err)
+}
+
+func (t *BucketManagerTest) TestSetUpBucketMethod2() {
```

Comment:
shall we rename it to something like TestSetUpBucketMethod_Multimount ?

Reply:
Good idea. Appended _IsMultiBucketMountTrue instead as discussed offline.